services/order.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+
+        my_order = Order.objects.create(
+            user=User.objects.get(username=username)
+        )
+
+        if date:
+            my_order.created_at = date
+            my_order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                order=my_order,
+                movie_session_id=ticket.get("movie_session"),
+                seat=ticket.get("seat"),
+                row=ticket.get("row")
+            )
+
+    return my_order
+
+
+def get_orders(username: str = None) -> QuerySet:

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]